a/amazon-metadata.yml

```diff
@@ -0,0 +1,19 @@
+amazon-metadata:
+  image: rancher/os-amazon-metadata:v0.9.2${SUFFIX}
```

Comment:
You've got me there - is there something that would trigger you to need to re-build?

if its only the parts within this image, then it can be a totally independent decision - or even date based - the basic pain is that if its the same as the RancherOS release, then we need to rebuild, or repush every release.